v0.5/mlperf.conf

```diff
@@ -3,6 +3,18 @@
 # Model maybe '*' as wildcard. In that case the value applies to all models.
 # All times are in milli seconds
 
+# set performance_sample_count for each model
+mobilenet.*.performance_sample_count_override = 1024
+resnet50.*.performance_sample_count_override = 1024
+ssd-mobilenet.*.performance_sample_count_override = 256
+ssd-resnet34.*performance_sample_count_override = 64
+gnmt.*.performance_sample_count_override = 3903900
```

Comment:
Will this cause an issue in AccuracyOnly mode? I thought we only need 3003 in AccuracyOnly mode.

Reply:
It should not because accuracy would go over the full dataset and not use PerformanceSampleCount() ... @briandersn true ?
I can give it a quick test to make sure.